assets/js/print.js

```diff
@@ -18,13 +18,22 @@
 
   $.printPreview = {
 
-    printPreview: function() {
+    printPreview: function(element) {
       var $body = $('body'),
           $container = $('.container-main'),
           footnoteLinks = "",
           linksIndex = 0;
 
       $body.find('.nav-mobile, .drilldown, .nav-main, .header-separator, .nav-service, .nav-lang, .form-search, .yamm--select, header > div:first-child, footer, .alert, .icon--print, .social-sharing, form, .nav-process, .carousel-indicators, .carousel-control, .breadcrumb, .pagination-container').remove();
+
+      // if an element is passed, we want it to be the only thing to print out
+      if (element) {
+        element = $('[data-print=' + element + ']');
+        var header = $('header');
+            title = element.attr('data-title') ? '<h1>' + element.attr('data-title') + '</h1>' : '';
```

Comment:
missing `var` before title. It's better declare the variable at the beginning of the function:

```javascript
var $body = $('body'),
     $container = /* ... */,
    header,
    title
;
    
```